.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 npm-debug.log
-tmp
\ No newline at end of file
+tmp
+.idea
```

Comment:
Can we remove this? This is better placed in a user's global ignore as it is specific to the user and not this project: `git config --global core.excludesfile .idea`